### DISCUSSION TITLE
Does Databend support infer / auto-detect a schema in SQL?

### DISCUSSION THREAD

**Discussion by bubbajoe** [databendlabs/databend#10634] [Q&A] [answered]
Hey, i want to know know if databend supports something like this: 
```
CREATE EXTERNAL TABLE taxi2 STORED AS PARQUET LOCATION 's3://bucket/files_*.parquet';
# example from Apache Arrow - Data Fusion 
```

..which allows you to quickly create a table without knowing the schema. Essentially inferring the schema.

Also does it work for: parquet + ndjson + csv?

**Comment by wubx** [comment, marked as answer]
>infer_schema currently only supports parquet file format.

https://databend.rs/doc/sql-functions/table-functions/infer_schema#syntax

**Reply by bubbajoe** [reply]
Ah thanks!
